### PR TITLE
WRKLDS-1319: A job that adds nodes on day2 in already created compact ipv4 cluster by ABI

### PR DESCRIPTION
**Pull request by pawanpinjarkar**
A job that adds nodes on day2 in already created compact ipv4 cluster by ABI

### CODE REVIEW

**Comment by pawanpinjarkar** [comment]
/pj-rehearse pull-ci-openshift-oc-master-e2e-agent-compact-ipv4-add-nodes

**Comment by pawanpinjarkar** [comment]
/pj-rehearse pull-ci-openshift-oc-master-e2e-agent-compact-ipv4-add-nodes

**Comment by pawanpinjarkar** [comment]
/pj-rehearse pull-ci-openshift-oc-master-e2e-agent-compact-ipv4-add-nodes

**Review by rwsu — COMMENTED** [review]
/lgtm

File: ci-operator/config/openshift/oc/openshift-oc-master.yaml [hunk 1]
```diff
@@ -194,6 +194,22 @@ tests:
   optional: true
   steps:
     workflow: openshift-ci-security
+- as: e2e-agent-compact-ipv4-add-nodes
+  cluster: build05
+  optional: true
+  run_if_changed: ^(cmd|pkg|data).*/agent/
```
Comment (ardaguclu) [inline ci-operator/config/openshift/oc/openshift-oc-master.yaml:200]: Should this run if `https://github.com/openshift/oc/tree/master/pkg/cli/admin/nodeimage` path (and as an optional also go.mod is changed maybe) is changed?.

**Review by rwsu — COMMENTED** [review]
/lgtm

**Pull request merged**